code/game/objects/items/stacks/medical.dm

```diff
@@ -241,7 +241,7 @@
 	amount = 15
 	max_amount = 15
 	repeating = TRUE
-	heal_brute = 10
+	heal_brute = 5
```

Comment:
Sutures already take ages to apply and you burn them very quickly. Why are we nerfing this and not stuff that needs it?